docs/helpers.md

```diff
@@ -232,7 +234,7 @@ class JSFailure extends codecept_helper {
   }
 
   _afterStep() {
-    if (this.err) throw new Error('Browser JS error '+this.err);
+    if (this.err) throw new Error('Browser JS error '+ this.err);
```

Comment:
why is the space after the plus and not before?

Reply:
it is just the format for better view, space is not appended to the string itself.

Reply:
:smile: Maybe add the space to before plus?

Reply:
@Vorobeyko I think using literal string will be better. :)